src/styles/components/collapsible-error-message.less

```diff
@@ -0,0 +1,54 @@
+
+//
+// The CollapsibleErrorMessage is a component used on top
+// of forms or on other places to display a main error
+// message and a floating drop-down with the error details
+//
+.collapsible-error-message {
+
+  //
+  // Shared styling between the two items
+  //
+  .collapsible-fixed, .collapsible-floating > div {
+    background: color-lighten(@red, 90);
+    padding: (@base-spacing-unit / 2) @base-spacing-unit;
```

Comment:
When we use `@base-spacing-unit`, we need to declare it for all of the screen sizes, like here: https://github.com/dcos/dcos-ui/blob/master/src/styles/components/dashboard-health-list.less#L49
(this file is not perfect, unfortunately)